examples/isomorphic/app/src/state/msg.rs

```diff
@@ -1,4 +1,7 @@
+use wasm_bindgen::JsValue;
+
 pub enum Msg {
     Click,
     SetPath(String),
+    StoreContributors(JsValue),
```

Comment:
I know the other two don't have comments (sorry!) but mind adding a comment to this new field?